src/dep/libs/lib7z/7zTypes.h

```diff
@@ -4,10 +4,6 @@
 #ifndef __7Z_TYPES_H
 #define __7Z_TYPES_H
 
-#ifdef __LIBRETRO__
-#include "streams/file_stream_transforms.h"
-#endif
-

```

Comment:
@barbudreadmon, is this ok?

Reply:
I grabbed the SDK from LZMA's website, unless you do not directly support the libretro-port then it should be ok, otherwise I will revert those changes.

Reply:
should be ok now